rules.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/styles.css" />
+    <title>Paths</title>
+  </head>
+  <body>
+    <div class="rules-container">
+      <nav class="quiz-navbar flex flex-align-center justify-between">
+        <div class="nav-logo flex justify-center">gehraaiyaan</div>
+        <div class="nav-right">
+          <!-- <button class="btn">
```

Comment:
remove unused code.

Reply:
Will remove them while refactoring. Thanks for reviewing.